spec/openapi.yml

```diff
@@ -1069,8 +1112,16 @@ components:
         args:
           description: The arguments of the run.
           type: object
+        facets:
+          $ref: '#/components/schemas/runFacets'
       example:
         args: {'email': 'me@example.com', 'emailOnFailure': false, 'emailOnRetry': true, 'retries': 1}
+        facets: {
+          'context': {
+            '_producer': 'https://github.com/OpenLineage/OpenLineage/blob/v1-0-0/client',
+            '_schemaURL': 'https://github.com/OpenLineage/OpenLineage/blob/v1-0-0/spec/OpenLineage.yml#MyCustomJobFacet'
```

Comment:
Would a job have a `_schemaURL`?

Reply:
this is an attribute of the facet. All facets have pointer to their schema

Reply:
Is this the schema for the job? I'm not understanding the  usage of `_schemaURL` here.

Reply:
The field `_schemaURL` points to the definition of the facet! Sorry, had to give the openlineage spec a quick review for clarity :)